rootfs-builder/alpine/Dockerfile.in

```diff
@@ -6,13 +6,36 @@
 From docker.io/golang:@GO_VERSION@-alpine
 
 RUN apk update && apk add \
-    git \
-    make \
+    apk-tools-static \
+    autoconf \
+    automake \
     bash \
+    binutils \
+    cmake \
+    coreutils \
+    curl \
+    g++ \
     gcc \
-    musl-dev \
-    linux-headers \
-    apk-tools-static \
+    git \
+    libc-dev \
     libseccomp \
     libseccomp-dev \
-    curl
+    linux-headers \
+    m4 \
+    make \
+    musl \
+    musl-dev \
+    tar \
+    vim
```

Comment:
Nit: really? But I prefer emacs! :smile: But seriously, do we need this package for all the distros?

Reply:
It's just build image. It might be better to have a editor to do some slight changes and then tests in build container. maybe

Reply:
We ideally want to keep the rootfs as minimal as possible keeping in check memory consumed by a container. So the recommendation would be to add a package only if it is required.